lib/widgets/icons.dart

```diff
@@ -206,6 +209,7 @@ IconData iconDataForStream(ZulipStream stream) {
   //   see this message and the one after it:
   //   https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/design.3A.20.23F117.20.22Inbox.22.20screen/near/1680637
   return switch(stream) {
+    ZulipStream(isArchived: true) => ZulipIcons.archive,
```

Comment:
Please don't start showing the icon in the same commit that adds the asset to the project. I mentioned this in my last review: https://github.com/zulip/zulip-flutter/pull/2037#pullrequestreview-3580155281